docs/G-Codes.md

```diff
@@ -642,14 +642,54 @@ section is enabled:
 
 ## Adxl345 Accelerometer Commands
 
-The following command is available when an "adxl345" config section is
+The following commands are available when an "adxl345" config section is
 enabled:
 - `ACCELEROMETER_MEASURE [CHIP=<config_name>] [RATE=<value>]
   [NAME=<value>]`: Starts accelerometer measurements at the requested
   number of samples per second. If CHIP is not specified it defaults
   to "default". Valid rates are 25, 50, 100, 200, 400, 800, 1600,
-  and 3200. If RATE is zero (or not specified) then the current series
-  of measurements are stopped and the results are written to a file
-  named `/tmp/adxl345-<name>.csv` where "<name>" is the optional NAME
-  parameter. If NAME is not specified it defaults to the current time
-  in "YYYYMMDD_HHMMSS" format.
+  and 3200. The command works in a start-stop mode: when executed for
+  the first time, it starts the measurements, next execution stops them.
+  If RATE is not specified, then the default value is used (either from
+  `printer.cfg` or `3200` default value). The results of measurements
+  are written to a file named `/tmp/adxl345-<name>.csv` where "<name>"
+  is the optional NAME parameter. If NAME is not specified it defaults
+  to the current time in "YYYYMMDD_HHMMSS" format.
+- `ACCELEROMETER_QUERY [CHIP=<config_name>] [RATE=<value>]`: queries
+  accelerometer for the current value. If CHIP is not specified it
+  defaults to "default". If RATE is not specified, the default value is
+  used. This command is useful to test the connection to the ADXL345
+  accelerometer: one of the returned values should be a free-fall
+  acceleration (+/- some noise of the chip).
+
+## Resonance Testing Commands
+
+The following commands are available when a "resonance_tester" config section
+is enabled:
+- `MEASURE_AXES_NOISE`: Measures and outputs the noise for all axes of all
+  enabled accelerometer chips.
+- `TEST_RESONANCES AXIS=<axis> OUTPUT=<resonances,raw_data>
+  [CSV_NAME=<csv_name>] [RAW_NAME=<raw_name>] [FREQ_START=<min_freq>]
```

Comment:
Very minor, but wouldn't it be simpler if there was just a single `NAME` parameter?  I can't see a case where a user would wish to use a "name" differently between the two.

Reply:
That's a good idea. Done.